qtawesome/iconic_font.py

```diff
@@ -26,7 +26,14 @@
                          QSizeF, QRectF, QPointF, QThread)
 from qtpy.QtGui import (QColor, QFont, QFontDatabase, QIcon, QIconEngine,
                         QPainter, QPixmap, QTransform, QPalette, QRawFont,
-                        QGlyphRun, QImage)
+                        QImage)
+try:
+    # Needed since `QGlyphRun` is not available for PySide2
+    # See spyder-ide/qtawesome#210
+    from qtpy.QtGui import QGlyphRun
+except ImportError:
+    QGlyphRun = None
+
 from qtpy.QtWidgets import QApplication
```

Comment:
```suggestion
from qtpy.QtWidgets import QApplication

try:
    # Needed since `QGlyphRun` is not available for PySide2
    # See spyder-ide/qtawesome#210
    from qtpy.QtGui import QGlyphRun
except ImportError:
    QGlyphRun = None
```

We accustom to leave try/except in imports after all imports have been declared.